text_handler.py

```diff
@@ -24,16 +22,20 @@ def __init__(self, bot: Bot):
         self.bot = bot
 
     async def handle(self, message: types.Message):
-        if not await self.__isProvideService(message):
+        if not await self.__is_provide_service(message):
             return
         text = message.text
-        if text and (' ' not in text and '\n' not in text) \
+        if text and (text.startswith("http")) \
+                and (' ' not in text and '\n' not in text) \
+                and not ('/sets/' in text and '?in=' not in text) \
                 and any(site in text for site in music_sites):
-            await self.__handleMusicGrabber(message)
+            await types.ChatActions.typing()
+            await self.__handle_music_grabber(message)
         elif text == 'ping':
-            await message.reply('pong', reply=False)
+            pong = await message.reply('pong', reply=False)
+            await self.__delete_messages(5, (pong, message))
 
-    async def __isProvideService(self, message: types.Message):
+    async def __is_provide_service(self, message: types.Message):
```

Comment:
```
Ilya the 大佬：
Although maybe you can define the function outside of the class but in the same file, and call it inside of the class

Dash Savage：
yeah, that's the solution
```

So did you try it? Or not yet

Reply:
not yet, so we are supposed to deal with most of `pylint` warns? then I think I have to change those no-self-use and no-else-return issue, will do it later or tomorrow.

Reply:
we need to look it up if it's really better to use **functions** instead of **methods**. I presume this lint check was added on purpose

Reply:
`will do it later or tomorrow.`
I think it's better not to do it in this pull request. Because this pull request is already pretty big, so if we're going to follow **no-self-use** it's better to merge this pull request and open a new one